adafruit_mpr121.py

```diff
@@ -30,8 +30,26 @@
 * Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
 """
 
+from __future__ import annotations
+
 import time
 
+try:
+    from typing import List, Optional, Tuple
+except ImportError:
+    # typing module does not exist in CircuitPython
+    class List(list):
+        """Typing hint indicating a list."""
+
+    # pylint: disable=too-few-public-methods
+    class Optional:
+        """Typing hint alias of ``Union[Any, None]``."""
+
+    class Tuple(tuple):
+        """Typing hint indicating a tuple."""
```

Comment:
I don't think we need to declare these fallback type classes. 

All of the typing information will get ignored by the CircuitPython interpreter so it's okay if it is unable to find these classes because it won't actually try to load them.